hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3571,6 +3571,19 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.client.server-defaults.validity.period.ms</name>
+    <tag>OZONE, CLIENT, SECURITY</tag>
+    <value>3600000</value>
```

Comment:
nit: we should add the default unit.

Reply:
And `.ms` should be omitted from the property name.